tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -290,7 +289,7 @@ def test_throughput_epm_hour_rollup(self):
             )
         assert response.status_code == 200, response.content
         data = response.data["data"]
-        assert len(data) == 7
+        assert len(data) == 6
```

Comment:
@evanh As the person that wrote this test, can you confirm if this is right to you?

Reply:
Yeah @evanh, can you confirm :)

Reply:
Confirmed 👍 😃 🤞 🎆